src/extensions/default/analyzers/Microsoft.DotNet.UpgradeAssistant.Extensions.Default.Analyzers.Common/AnalysisContextExtensions.cs

```diff
@@ -0,0 +1,35 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.Diagnostics;
+
+using CS = Microsoft.CodeAnalysis.CSharp;
+using VB = Microsoft.CodeAnalysis.VisualBasic;
+
+namespace Microsoft.DotNet.UpgradeAssistant.Extensions.Default.Analyzers.Common
+{
+    public static class AnalysisContextExtensions
+    {
+        public static void RegisterSimpleMemberAccessExpression(this AnalysisContext analysisContext, Action<SyntaxNodeAnalysisContext> analyze, Action<SyntaxNodeAnalysisContext>? vbAnalyze = null)
```

Comment:
I like this. Maybe the Action can also take in the string identifiers so that's readily available as well without needing to know about vb/cs?